### PR TITLE
Add inline demo & update API keys

### PR DESCRIPTION
**Pull request by samuelli**
Preview link: https://beta.webcomponents.org/preview/samuelli/google-map/0af17556f9a49bc5602bd856bc2740ef3cc21a2f

### CODE REVIEW

**Comment by samuelli** [comment]
cc/ @ebidel

File: README.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
 ==========
 
-See https://elements.polymer-project.org/elements/google-map
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://beta.webcomponents.org/element/GoogleWebComponents/google-map)
 
+<!---
+```
+<custom-element-demo>
+  <template>
+    <script src="../webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (ebidel) [inline README.md]: can this be the webcomponents-lite.min.js file or will that mess up the inline demo? We should encourage folks to use that.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+    <link rel="import" href="google-map.html">
+    <style is="custom-style">
```
Comment (ebidel) [inline README.md]: remove`is="custom-style"`?

File: README.md (hunk 1 above)
Comment (samuelli) [inline README.md]: Sure, done

**Comment by samuelli** [comment]
Updated: https://beta.webcomponents.org/preview/samuelli/google-map/98703b21d4e2fa2fff95e4752bb0b8e4794ecf64

If you enable preview-integration, the preview links get auto-generated :)

**Comment by ebidel** [comment]
Do we need to update the release for the catalog to pick it up?

**Comment by samuelli** [comment]
@ebidel: yep.

**Pull request merged**